static/app/components/overlayArrow.tsx

```diff
@@ -68,11 +68,11 @@ function ChonkOverlayArrow({
 }
 
 const ChonkWrap = chonkStyled('div')<{
-  size: number;
+  dimensions: number;
```

Comment:
note: I renamed this to `dimensions` because `size` is a native prop that doesn’t get filtered out by emotion, so we had a `<div size=“16”>` in the dom.